src/app/lib/analyticsUtils/index.js

```diff
@@ -295,4 +301,36 @@ export const getThingAttributes = (attribute, articleData) => {
   return null;
 };
 
+export const getCampaignType = () => {
+  if (!onClient()) return null;
```

Comment:
When were we considering getting rid of the `onClient` check, in one of the other issues or a future enhancement?

Reply:
A future enhancement as it covers most functions in this file which are out of scope of this PR